explorer/src/Pos/Explorer/Web/Api.hs

```diff
@@ -114,6 +114,14 @@ data ExplorerApiRecord route = ExplorerApiRecord
         :> Capture "address" CAddress
         :> ExRes Get CAddressSummary
 
+  , _addressUtxoBulk :: route
```

Comment:
I don't like the idea that you have to do something in "bulk" - while it may be sensible, using a single address to return a utxo seems like a way to go since it would restrain the users to not use 10500 addresses and effectively kill the explorer node.

Reply:
In this case, as I stated in the PR description above, it was more a matter of need for my use case (a light wallet with multiple addresses) given the current performance limitations of the database when fetching utxo for a single address, but I believe that it would be generally useful to have a bulk API for addresses. E.g. getting a paginated chronologically ordered transaction history for multiple addresses would be quite cumbersome without a bulk API, which is a different issue of course, but I had it in mind when adding this first bulk endpoint.